lib/controllers/quotes.js

```diff
@@ -1,3 +1,18 @@
 const { Router } = require('express');
+const { Quote } = require('../models/Quote');
+const router = Router();
 
-module.exports = Router();
+router
+  .post('/', async (req, res, next) => {
+    try {
+      const quote = await Quote.insert(req.body);
+      if (req.body.episode_id) {
+        await Promise.all(req.body.episode_id.map(id => quote.addQuoteByID(id)));
```

Comment:
There is actually no need for a promise.all() here, because we are only working on inserting one row of data, making a single promise.